config/7.1.0/securebanking/ig/scripts/groovy/RepoConsent.groovy

```diff
@@ -109,61 +109,14 @@ def convertIDMResponse(intentResponseObject, intentType) {
                     "oauth2ClientName"     : intentResponseObject.apiClient.name
             ]
             break
-        case IntentType.PAYMENT_DOMESTIC_CONSENT:
+        else: // For all payments intents
```

Comment:
Please test this, the syntax looks wrong to me as you still have a switch statement but now there is an else block?

Reply:
I don't know in what I was thinking when I added an `else` as case instead of `default`... Fixed.